ModularTegustation/fishing/code/fishcooking/fishfood.dm

```diff
@@ -0,0 +1,98 @@
+//Grilled Fish
+
+/obj/item/food/cooked_fish
+	name = "cooked fish"
+	desc = "A unbutchered fish thats been cooked."
```

Comment:
Unbutchered is such a weird word. If the fish has not been gutted, it is a whole fish.